src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/Batching/CashoutsBatchAggregate.cs

```diff
@@ -130,11 +130,14 @@ public static Guid GetNextId()
             return Guid.NewGuid();
         }
 
-        public TransitionResult AddCashout(Guid cashoutId, Guid clientId, string toAddress, decimal amount)
+        public TransitionResult AddCashout(BatchedCashoutValueType cashout)
         {
             var oldCashoutsCount = Cashouts.Count;
 
-            Cashouts.Add(new BatchedCashoutValueType(cashoutId, clientId, toAddress, amount));
+            if (oldCashoutsCount < CountThreshold)
+            {
+                Cashouts.Add(cashout);
```

Comment:
Suggest to move:
`LastCashoutAdditionMoment = DateTime.UtcNow;` 
just after the 
`Cashouts.Add(cashout);` line 
and remove condition at 170